SEQLinkage/Main.py

```diff
@@ -28,16 +28,16 @@ class Args:
     def __init__(self):
         self.parser = ArgumentParser(
         description = '''\t{}, linkage analysis using sequence data\n\t[{}]'''.\
-        format(env.proj, env.version),
+        format("SEQLinkage", VERSION),
         formatter_class = RawDescriptionHelpFormatter,
-        prog = env.prog,
+        prog = 'seqlink',
         fromfile_prefix_chars = '@', add_help = False,
         epilog = '''\tCopyright (c) 2013 - 2014 Gao Wang <wang.gao@columbia.edu>\n\tDistributed under GNU General Public License\n\tHome page: {}'''.format(HOMEPAGE))
         self.getEncoderArguments(self.parser)
         self.getIOArguments(self.parser)
         self.getLinkageArguments(self.parser)
         self.getRuntimeArguments(self.parser)
-        self.parser.set_defaults(func=main)
+        #self.parser.set_defaults(func=main)
```

Comment:
I cannot remember what this does, but it seems necessary to me? If it is truly unnecessary you should remove it and not comment it out?

Reply:
make install easier with pip